src/main/java/org/jenkinsci/plugins/workflow/steps/StepDescriptor.java

```diff
@@ -93,6 +104,79 @@ public boolean isAdvanced() {
     }
 
     /**
+     * Some steps, such as {@code CoreStep} or {@code GenericSCMStep} can take
+     * arbitrary {@link Describable}s of a certain type and execute it as a step.
+     * Such a step should return true from this method so that {@link Describable}s that
+     * it supports can be directly written as a step as a short-hand.
+     *
+     * <p>
+     * Meta-step works as an invisible adapter that creates an illusion that {@link Describable}s are
+     * steps.
+     *
+     * <p>
+     * For example, in Jenkins Pipeline, if there is a meta step that can handle a {@link Describable},
+     * and it has a symbol, it allows the following short-hand:
+     *
+     * <pre>
+     * public class Xyz extends Foo {
+     *     &#64;DataBoundConstructor
```

Comment:
For anyone interested http://stackoverflow.com/questions/541920/multiple-line-code-example-in-javadoc-comment)